templates/core/terraform/scripts/letsencrypt.sh

```diff
@@ -3,6 +3,8 @@ set -e
 
 script_dir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 
+echo $FQDN
```

Comment:
Is this debug?

Reply:
Yes it is there should have been no changes to this file, Will remove now